### PR TITLE
Feature/keep alive

### PR DESCRIPTION
**Pull request by milanzlicic**

### CODE REVIEW

**Review by svranesevic — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -179,3 +179,22 @@ std::unique_ptr<wolkabout::Wolk> wolk =
 
     wolk->connect();
 ```
+
+**Ping Mechanism:**
```
Comment (svranesevic) [inline README.md]: Custom keep alive mechanism

File: README.md [hunk 2]
```diff
@@ -179,3 +179,22 @@ std::unique_ptr<wolkabout::Wolk> wolk =
...
+
+WolkAbout C++ Connector by default uses ping mechanism to notify Wolkabout IOT Platform that device is still connected.
```
Comment (svranesevic) [inline README.md]: **WolkAbout IoT**

File: README.md [hunk 3]
```diff
@@ -179,3 +179,22 @@ std::unique_ptr<wolkabout::Wolk> wolk =
...
+Ping message is sent to Wolkabout IOT Platform every 10 minutes.
```
Comment (svranesevic) [inline README.md]: **WolkAbout IoT**

File: README.md [hunk 4]
```diff
@@ -179,3 +179,22 @@ std::unique_ptr<wolkabout::Wolk> wolk =
...
+
+To reduce network usage ping mechanism can be disabled in following manner:
+
+```cpp
+wolkabout::Device device("DEVICE_KEY", "DEVICE_PASSWORD", {"ACTUATOR_REFERENCE_ONE", "ACTUATOR_REFERENCE_TWO"});
+
+std::unique_ptr<wolkabout::Wolk> wolk =
+  wolkabout::Wolk::newBuilder(device)
+    // actuation handlers setup
+    .withoutInternalPing() // Disable ping mechanism
```
Comment (svranesevic) [inline README.md]: withoutKeepAlive

**Review by svranesevic — CHANGES_REQUESTED** [review]

**Review by svranesevic — DISMISSED** [review]

**Review by srdjastankovic — APPROVED** [review]

**Pull request merged**